resources/js/components/home-footer.js

```diff
@@ -278,8 +291,37 @@ class HomeFooter extends LitElement {
         return /^(https?:\/\/|data:image|\/|\.\/|\.\.\/)/.test(icon)
     }
 
+    reset_apps() {
+        const confirmDelete = confirm(
+            'Are you sure you want to reset all apps?'
+        )
+
+        if (confirmDelete) {
+            fetch(this.appUrl + '/reset-apps', {
```

Comment:
@Perumal605 , Please use the magic_url() helper.

Reply:
@incraigulous Now I am using the Magic_url helper.